corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -120,20 +120,7 @@ def update_sync_log_with_checks(sync_log, xform, cases, case_db,
         sync_log.update_phone_lists(xform, cases)
     except SyncLogAssertionError, e:
```

Comment:
@czue Ok so probably nothing tests this code; if anything did, the below would just get us into an infinite loop, since it just re-calls the same function with the same state/args.

I'm thinking maybe the best thing is to... just let `SyncLogAssertionError` bubble up to the top and 500? Wouldn't that keep the submission on the phone, and then they could re-sync their phone and that may fix some of these assertions